src/index.jsx

```diff
@@ -112,18 +103,22 @@ export default class MultiselectTwoSides extends React.Component {
 	}
 
 	handleClickDeselectAll() {
-		this.props.onChange([]);
+		const {options, onChange, valueKey, value} = this.props;
```

Comment:
This is the actual change, instead of cleaning up all the values, it will keep the ones that are already "selected" and have the `disabled` attribute.